requirements.txt

```diff
@@ -1,3 +1,3 @@
 # basic
-setuptools==40.0.0
+setuptools==52.0.0
```

Comment:
Due to this line, the dev environment uses `setuptools` 52.0.0 but the MSI uses the latest, causing inconsistent behavior (https://github.com/Azure/azure-cli/pull/24982#issuecomment-1369405440).